flyteadmin/pkg/common/filters.go

```diff
@@ -37,6 +38,8 @@ const (
 	joinArgsFormat          = "%s.%s"
 	containsQuery           = "%s LIKE ?"
 	containsArgs            = "%%%s%%"
+	notContainsQuery        = "%s NOT LIKE ?"
```

Comment:
nit: not contains read a bit weird, why not 'NotLike' here? Then we could even support regex or wild carding with this too

Reply:
good idea. Then we can filter by start with. Updated.